src/board_controller/openbci/ganglion_bglib/callbacks.cpp

```diff
@@ -93,7 +93,7 @@ void ble_evt_connection_disconnected (const struct ble_msg_connection_disconnect
 {
     // atempt to reconnect
     ble_cmd_gap_connect_direct (
-        &GanglionLib::connect_addr, gap_address_type_random, 40, 60, 100, 0);
+        &GanglionLib::connect_addr, gap_address_type_random, 10, 76, 100, 0);
```

Comment:
Could you add a comment indicating what these values represent, and how you figured out the right numbers? Even if the comment is "I don't know why". Also would be good to leave a comment indicating that if these values  are wrong, packet loss occurts.